### PR TITLE
MML-190 tooltip added

### PR DESCRIPTION
**Pull request by symphony-enrico**
See https://github.com/symphonyoss/messageml-utils/issues/190

### CODE REVIEW

File: src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java [hunk 1]
```diff
@@ -389,6 +389,12 @@ public Element createElement(org.w3c.dom.Element element, Element parent) throws
         }
 
       case Div.MESSAGEML_TAG:
+        String clazz = element.getAttribute(CLASS_ATTR);
+        if(SplittableElement.SPLITTABLE_GROUPS.contains(clazz)){
+          // Special div created by a splittable element, it is not converted in MessageML
+          // as it exist only in PresentationML
+          return null;
```
Comment (symphony-enrico) [inline src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java:396]: When converting from PresentationML to MessageML, in some cases there is a div wrapper to remove:

```xml
<div class="element-implementing-splittable-group">
...
   <presentationMLelement>value</presentationMLelement>
 </div>
```

```xml
<messageMLelement attribute=...>value</messageMLelement >
```

The divs to be dropped are identified by reading the class attribute ("textfield-group", "textarea-group", "dropdown-group", "person-selector-group")
The problem is that this list must be maintained and it is not so reliable. A suggest a specification modification for
the div wrapper in presentationML, by adding a very specific attribute

File: src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java [hunk 2]
```diff
@@ -169,6 +167,17 @@ private void buildElement(MessageMLParser context, org.w3c.dom.Element element)
       }
 
       addChild(child);
+    } else if(element.getNodeName().equals(Div.MESSAGEML_TAG)) {
+      /*
+      When converting from PresentationML -> MessageML some elements are not converted
```
Comment (symphony-enrico) [inline src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java:172]: In reality the conversion it is not PresentationML to MessageML, but to **MessageML tree object** (that can be in PresentationML or MessageML format)

File: src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java [hunk 3]
```diff
@@ -470,15 +535,72 @@ public void loadElementId(String id) throws InvalidInputException {
...
+   */
+  public Optional<Map<Class<? extends SplittableElement>, Pair<String, String>>> getAllSplittableValues(String id) throws InvalidInputException {
+    SplittableData data = splittableComponents.get(id);
+    if(data == null){
+      return Optional.empty();
+    }
+    return Optional.of(data.getAllValues());
+  }
+
+  private void createSpittable(String id, Class<? extends SplittableElement> clazz, Pair<String,String> value)
```
Comment (lmacedodaitan) [inline src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java:573]: Typo in the method's name.

Comment (symphony-enrico) [reply src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java]: Hi @lmacedodaitan I fixed this...and BTW another small bug, missing data-generated="true" attribute when converting to ProsentationML

**Pull request merged**